src/block/Fungus.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\block;
+
+use pocketmine\item\Item;
+use pocketmine\math\Facing;
+use pocketmine\math\Vector3;
+use pocketmine\player\Player;
+use pocketmine\world\BlockTransaction;
+
+class Fungus extends Flowable{
+
+	public function ticksRandomly() : bool{
+		return true;
+	}
+
+	public function onNearbyBlockChange() : void{
+		if($this->getSide(Facing::DOWN)->isTransparent()){
+			$this->position->getWorld()->useBreakOn($this->position);
+		}
+	}
+
+	public function place(BlockTransaction $tx, Item $item, Block $blockReplace, Block $blockClicked, int $face, Vector3 $clickVector, ?Player $player = null) : bool{
+		$down = $this->getSide(Facing::DOWN);
+		$downId = $down->getTypeId();
+		if((!$down->isTransparent()) || $downId === BlockTypeIds::MYCELIUM || $downId === BlockTypeIds::PODZOL || $downId === BlockTypeIds::CRIMSON_NYLIUM || $downId === BlockTypeIds::WARPED_NYLIUM || $downId === BlockTypeIds::FLOWER_POT){
```

Comment:
Unnecessary parentheses
```suggestion
		if(!$down->isTransparent() || $downId === BlockTypeIds::MYCELIUM || $downId === BlockTypeIds::PODZOL || $downId === BlockTypeIds::CRIMSON_NYLIUM || $downId === BlockTypeIds::WARPED_NYLIUM || $downId === BlockTypeIds::FLOWER_POT){
```

Reply:
Would not be better to create a tag for that ?

Reply:
You mean something like in NetherRoots?

Reply:
Like mud or dirt to detect which block can make an exception for a "mushroom" like block

Reply:
Okay when I am home, I will look at this

Reply:
@ShockedPlot7560 

now i use the "tags", but i don't know if that's what you wanted.